### PR TITLE
Add Investing style page UI

### PR DESCRIPTION
**Pull request by manhduc-nguyen**
Screen Shot 2022-01-18 at 17 42 26

### CODE REVIEW

File: lib/app/cubit/intro/intro_cubit.dart [hunk 1]
```diff
@@ -9,10 +9,10 @@ class IntroCubit extends Cubit<IntroState> {
   IntroCubit() : super(IntroUninitState());
 
   dynamic goToLogInScreen() =>
-      navigatorKey.currentState?.pushNamed(BeanstalkPageName.log_in);
+      navigatorKey.currentState?.pushNamed(BeanstalkPageName.link_funds);
 
   dynamic goToSignUpScreen() =>
-      navigatorKey.currentState?.pushNamed(BeanstalkPageName.sign_up);
+      navigatorKey.currentState?.pushNamed(BeanstalkPageName.link_funds);
```
Comment (mduccc) [inline lib/app/cubit/intro/intro_cubit.dart:15]: We only need one entry point to go to the `link_funds` screen

File: lib/app/designsystem/ds_colors.dart [hunk 2]
```diff
@@ -17,4 +17,6 @@ class DSColors {
   static const Color fiYellow = Color(0xFFCFFF00);
   static const Color coffee = Color(0xFF2D2D29);
   static const Color stone = Color(0xFFE0ECE6);
+  static const Color moss = Color(0xFF94C6A4);
+  static const Color mossOpacity10 = Color(0x1A94C6A4);
```
Comment (mduccc) [inline lib/app/designsystem/ds_colors.dart:21]: Consider use `moss.withOpacity(0.1)` instead `mossOpacity10`.
I think should only put original hex color in `ds_color`

File: lib/app/page/investing_style/investing_style_page.dart [hunk 3]
```diff
@@ -10,6 +13,193 @@ class InvestingStylePage extends StatefulWidget {
...
+            )),
+        Positioned.fill(
+            top: 130,
+            child: SingleChildScrollView(
+              child: Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 20.0),
+                child: Column(
+                  children: [
+                    Container(
+                      width: double.maxFinite,
```
Comment (mduccc) [inline lib/app/page/investing_style/investing_style_page.dart:51]: Consider separate the widget in the future

**Review by mduccc — APPROVED** [review]

File: lib/app/cubit/intro/intro_cubit.dart (hunk 1 above)
Comment (manhduc-nguyen) [reply lib/app/cubit/intro/intro_cubit.dart:15]: its pointing to a black screen right now, u can change it to the correct one when working with signup screen, thats better.

File: lib/app/designsystem/ds_colors.dart (hunk 2 above)
Comment (manhduc-nguyen) [reply lib/app/designsystem/ds_colors.dart:21]: agreed

**Pull request merged**